[BE] Fix SIM105 `suppressible-exception`

justinchuby opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #100336
* #100342
* #100341
* #100338
* #100339

Use contextlib.suppress({exception}) instead of try-except-pass

Signed-off-by: Justin Chu <justinchu@microsoft.com>



cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @ngimel @yf225 @chenyang78 @anijain2305 @soumith @desertfire

justinchuby committed:
[BE] Fix SIM105

Signed-off-by: Justin Chu <justinchu@microsoft.com>

[ghstack-poisoned]

Skylion007 commented:
This change is one of the more controversial of Flake8-Simplify. lLst I checked, it can actually be slower than the previous approach and it not clearly more readable (the "zero-cost" exception handling in 3.11 may have changed that performance degradation though).

justinchuby committed:
Update on "[BE] Fix SIM105 `suppressible-exception`"


Use contextlib.suppress({exception}) instead of try-except-pass

Signed-off-by: Justin Chu <justinchumicrosoft.com>



cc soumith voznesenskym penguinwu anijain2305 EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 desertfire

[ghstack-poisoned]

justinchuby committed:
Update on "[BE] Fix SIM105 `suppressible-exception`"


Use contextlib.suppress({exception}) instead of try-except-pass

Signed-off-by: Justin Chu <justinchumicrosoft.com>



cc soumith voznesenskym penguinwu anijain2305 EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 desertfire

[ghstack-poisoned]

justinchuby committed:
Update on "[BE] Fix SIM105 `suppressible-exception`"


Use contextlib.suppress({exception}) instead of try-except-pass

Signed-off-by: Justin Chu <justinchumicrosoft.com>



cc soumith voznesenskym penguinwu anijain2305 EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 desertfire

[ghstack-poisoned]

justinchuby committed:
Update on "[BE] Fix SIM105 `suppressible-exception`"


Use contextlib.suppress({exception}) instead of try-except-pass

Signed-off-by: Justin Chu <justinchumicrosoft.com>



cc soumith voznesenskym penguinwu anijain2305 EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 desertfire

[ghstack-poisoned]

justinchuby committed:
Update on "[BE] Fix SIM105 `suppressible-exception`"


Use contextlib.suppress({exception}) instead of try-except-pass

Signed-off-by: Justin Chu <justinchumicrosoft.com>



cc soumith voznesenskym penguinwu anijain2305 EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 desertfire

[ghstack-poisoned]

justinchuby committed:
Update on "[BE] Fix SIM105 `suppressible-exception`"


Use contextlib.suppress({exception}) instead of try-except-pass

Signed-off-by: Justin Chu <justinchumicrosoft.com>



cc soumith voznesenskym penguinwu anijain2305 EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng Xia-Weiwen wenzhe-nrv jiayisunx peterbell10 desertfire

[ghstack-poisoned]
